File: Tribonacci sequence.py
def tribonacci(signature, n):
    #your code here
    sig = signature
    a = sig[0]
    b = sig[1]
    c = sig[2]
    count = 0
    new = []
    if n == 0:
        return []
    else:
        while count < n:
            new.append(a)
            d = a + b + c
            a = b
            b = c
            c = d
            count += 1
    return new

File: Tribonacci sequence.js
function tribonacci(signature,n){
  //your code here
  var sig = signature;
  var a = sig[0];
  var b = sig[1];
  var c = sig[2];
  var result = [];
  var count = 0;
  
  if(n == 0) {
    return []
  } else {
    while (count < n) {
      result.push(a);
      var d = a + b + c;
      a = b
      b = c
      c = d
      count++;
    }
  }
  return result
}
 Correctamundo! You ma
